.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
+
+on: push
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - name: Clone
+        uses: actions/checkout@v4
+
+      - name: Build
+        run: dotnet build
+
+      # These tests don't work without sample data. Should we do anything about it?
```

Comment:
Tests have always been a bit of an issue.  Since they are copyrighted files, I don't want to include them in the repo.  I should tag them as Integration tests with a Category, and set up the tests to exclude them.  I know that is possible with NUnit. Is there a way to do it with MS Test?

Reply:
I have added this functionality. only Unit tests will be executed in the script.